python/plugins/processing/gui/DestinationSelectionPanel.py

```diff
@@ -267,8 +269,12 @@ def selectDirectory(self):
             settings.setValue('/Processing/LastOutputPath', dirName)
 
     def setValue(self, value):
-        if value == 'memory:':
+        if value == 'memory:' or not value:
```

Comment:
Again, I think this will break handling of optional outputs which are set to "skip output"